src/BloomExe/Publish/Epub/ElectronicPublishView.cs

```diff
@@ -58,6 +58,10 @@ public EpubView SetupEpubControl(EpubView view, NavigationIsolator _isolator, Ac
 			{
 				view = new EpubView();
 				_epubPreviewBrowser = new Browser();
+				// We rather mangled the Readium code in the process of cutting away its own navigation
+				// and other controls. It produces all kinds of JavaScript errors, but it seems to do
+				// what we want. So just suppress the toasts for all of them.
+				_epubPreviewBrowser.SuppressJavaScriptErrors = true;
```

Comment:
I'm ok with this, but just wanted to confirm that you'd rather have this than have false iff in Debug.